Modules/Axis/Sources/Axis/String/Collection.swift

```diff
@@ -0,0 +1,16 @@
+extension Collection where Iterator.Element: Equatable, Iterator.Element == SubSequence.Iterator.Element {
+    func index(of part: Self) -> Index? {
```

Comment:
No, this is something I came up with that would employ the power of default protocol implementations _and_ still fit the initial API that was there for `CharacterView`.

Sure, I will create some UT.